pages/portfolio/[id].js

```diff
@@ -23,29 +23,34 @@ export default function Content() {
             <div>
             <div className='container_main'>
                 <div className='row'>
-                    <div className='col-xl-6 col-lg-6 col-md-12 col-sm-12 '>
+                    <div className='col-xl-6 col-lg-6 col-md-6 col-sm-12 '>
                         <div className='text_container text-center col-xl-12'> 
                             <h1 className='txt_mainTitle'> {p_content.name} </h1>
                             <p className='txt_description'> {p_content.desc1} </p>
                             <a className='btn_visitSite' href={p_content.site} target = "_blank" rel = "noreferrer">Visit Site</a>  
                         </div>
                     </div>
-                    <div className='col-xl-3 col-lg-3 col-md-12 col-sm-12 '>
+                    <div className='col-xl-3 col-lg-3 col-md-6 col-sm-12 '>
                         <div className='text_container text-center'> 
                             <img className = "img-responsive" src={p_content.image1}></img>
                         </div>
                     </div>
                 </div>
-                <div className="container about-container">
-                {/*<div className="about-text">
-                    <p className="p1">WHAT WE DO</p>
-                    <p className="p2">{p_content.desc2}</p>                       
-                </div>*/}
+                <div className="about-container">
+                  <div className="about-text">
+                      <p className="p1">WHAT WE DO</p>
+                      <p className="p2">{p_content.tech1}</p>     
+                      <p className="p2">{p_content.tech2}</p> 
+                      <p className="p2">{p_content.tech3}</p> 
+                      <p className="p2">{p_content.tech4}</p>    
+                      <p className="p2">{p_content.tech5}</p>       
```

Comment:
It would be better if we iterate over all the items in the array.
```suggestion
                      {p_content.tech.map((tech) => (
                         <p className="p2">{tech}</p>    
                      )}                       
```

Reply:
noted, sir! nilagyan ko na rin po ng key prop